### PR TITLE
add \includeonly completion

### PR DESCRIPTION
**Pull request by Jieli12**
add \includeonly completion

### CODE REVIEW

File: src/providers/completion.ts [hunk 1]
```diff
@@ -155,7 +155,7 @@ export class Completer implements vscode.CompletionItemProvider {
                 break
 
             case 'input':
-                reg = /\\(input|include|subfile|includegraphics|lstinputlisting|verbatiminput)\*?(?:\[[^[\]]*\])*{([^}]*)$/
+                reg = /\\(input|include|subfile|includegraphics|lstinputlisting|verbatiminput|includeonly)\*?(?:\[[^[\]]*\])*{([^}]*)$/
```
Comment (jlelong) [inline src/providers/completion.ts:158]: Full support for `includeonly` is a bit more complicated that as it expects a coma separated list of files and files containing a coma in their names should be enclosed into `{}`

Comment (Jieli12) [reply src/providers/completion.ts:158]: Yes, I noticed that. The items after comma can not be completed when one finished first completion. Would you please add this feature? I am not expert in this field.

Comment (jlelong) [reply src/providers/completion.ts:158]: I have proposed a new implementation of this feature in #2088. So, I am closing this PR

Comment (Jieli12) [reply src/providers/completion.ts:158]: Great! Thank you very much!

**Pull request closed without merging**